### PR TITLE
JSTL - instructorHome

### PR DESCRIPTION
**Pull request by yj-soh**
The beginning.

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/instructor/copyModal.tag [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                    <input type="submit" class="btn btn-primary" id="fscopy_submit" value="Copy">
+                    <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
+               </div>
+            </form>
+        </div>
+    </div>
+</div>
```
Comment (damithc) [inline src/main/webapp/WEB-INF/tags/instructor/copyModal.tag]: should we have separate folder for each page?
```
tags/instructor (for tags common to instructor pages)
tags/instructorHome (for page specific tags)
```

File: src/test/resources/pages/InstructorHomeHTML.html [hunk 2]
```diff
@@ -1,241 +1,318 @@
 <div id="frameBodyWrapper" class="container theme-showcase">
```
Comment (damithc) [inline src/test/resources/pages/InstructorHomeHTML.html]: There should be no changes to html pages. If html pages have to change, we need to do that change first as a separate issue before converting to jstl.

File: src/main/webapp/WEB-INF/tags/instructor/copyModal.tag (hunk 1 above)
Comment (yj-soh) [inline src/main/webapp/WEB-INF/tags/instructor/copyModal.tag]: I believe it should be good to use subfolders instead.
As in, `tags/instructor/home`.
[It appears to be possible to combine them into a single prefix.](http://stackoverflow.com/questions/10592434/jsp-tag-files-in-subdirectories-using-a-single-taglib-prefix-is-that-possible)

**Comment by wkurniawan07** [comment]
different
Top = this PR, bottom = master. Is this going to be fixed?

**Comment by MrFool** [comment]
We will need to create a different template for this, open an issue and let's keep it in view to fix

**Pull request merged**